sdk/digitaltwins/digital-twins-core/CHANGELOG.md

```diff
@@ -1,17 +1,14 @@
 # Release History
 
-## 1.1.0 (unreleased)
+## 1.1.0-beta.1 (2022-04-04)
 
 ### Features Added
 
+- This release is a preview targeting the new data-plane api version: 2021-06-30-preview.
```

Comment:
I would suggest not mentioning the API version, but instead the new features that come with it (sourceTime in our case).